opentelemetry/src/metrics/instruments/counter.rs

```diff
@@ -5,6 +5,8 @@ use std::sync::Arc;
 use super::SyncInstrument;
 
 /// An instrument that records increasing values.
+///
+/// [`Counter`] can be cloned to create multiple handles to the same instrument. Avoid creating duplicate [`Counter`]s for the same instrument.
```

Comment:
```suggestion
/// [`Counter`] can be cloned to create multiple handles to the same instrument. Avoid creating duplicate [`Counter`]s for the same metric.
```

Is this better?
Also, should we put a stronger statement about re-using this struct, instead of repeatedily created it over and over?

Reply:
> Is this better?

Yes. I'll update them to use the term "metric".

> Also, should we put a stronger statement about re-using this struct, instead of repeatedily created it over and over?

We should do that if it's super bad for perf.  But I was thinking since we cache the aggregators and there is no increased contention for creating duplicate instruments (unlike .NET), this level of warning is enough?

Reply:
its best to have that warning now itself, so that even if a future implementation change causes slower perf when re-creating existing metric, we have it covered by doc warnings.

Reply:
Okay. How does this look?

```text
/// [`Counter`] can be cloned to create multiple handles to the same instrument. If a [`Counter`] needs to be shared, 
users are recommended to clone the [`Counter`] instead of creating duplicate [`Counter`]s for the same metric. Creating 
duplicate [`Counter`]s for the same metric could lower SDK performance.
```

Reply:
its good 👍